src/blockrelay/graphene_set.cpp

```diff
@@ -84,7 +88,19 @@ CGrapheneSet::CGrapheneSet(size_t _nReceiverUniverseItems,
 
     // Construct IBLT
     uint64_t nIbltCells = std::max((int)IBLT_CELL_MINIMUM, (int)ceil(optSymDiff));
-    pSetIblt = std::make_shared<CIblt>(CIblt(nIbltCells, ibltSalt, version >= 2));
+    uint8_t nCheckSumBits;
+    if (ibltVersion >= 2)
+    {
+        nCheckSumBits = CGrapheneSet::NChecksumBits(nIbltCells * CIblt::OptimalOverhead(nIbltCells),
+            CIblt::OptimalNHash(nIbltCells), nReceiverUniverseItems, fpr, UNCHECKED_ERROR_TOL);
+    }
+    else
+        nCheckSumBits = MAX_CHECKSUM_BITS;
+
+    LOG(GRAPHENE, "using %d checksum bits in IBLT\n", nCheckSumBits);
+    uint32_t keycheckMask = 0xffffffff >> (MAX_CHECKSUM_BITS - nCheckSumBits);
```

Comment:
I suggest to make, for completeness, another constant for the mask of `0xffffffff`, such as `MAX_CHECKSUM_MASK` or so. Goes for the other places where that value is used like that as well.

Reply:
Done.